content/outcomes/secure-software-supply-chain/establish-capability/establish-devsecops.md

```diff
@@ -0,0 +1,108 @@
+a development team writes the code, a security team ensures that this code
+adheres to organizational compliance requirements, and operations teams
+maintained the availability of the developed service. While this organization of
+specific roles often met the demand for bringing code to production, it suffers
+from a number of drawbacks.
+
+First, it was often slow. With different groups all involved in bringing code to
+production, responsibilities were often serialized. Once the code is ready, the
+security team takes a look. And then, once the security team blesses the
+release, an operator can deploy and maintain it. Yes, these teams could
+multi-task, and begin the next body of work while their current release
+candidate was being validated. But, this is typically not an efficient process.
+In practical terms, it introduces disruptive context switching, but it also
+means that these teams are not working collaboratively towards a common goal.
+
+Because these teams each have different objectives, friction often develops when
+attempting to achieve outcomes. Development teams are often blindsided by
+unclear or poorly understood security requirements, operations teams do not
+usually have nuanced understanding of the code to effectively operate it, and
+security teams struggle to keep the business secure in the face of rapidly
+evolving technical requirements.
+
+While Secure Software Supply Chains are primarily a component of getting code
+into production, they are also an opportunity to reduce the friction between the
+various stakeholders attached to software development efforts. When implemented
+properly, each part of the organization will be working together towards the
+common goal of delivering customer value through software delivery.
+
+## Changing Responsibilities
+
+Members of a development organization will still maintain their respective areas
+of expertise, but they will now be asked to work more collaboratively towards
+this common goal of bringing code to production. Instead of serialized tasks,
+within DevSecOps each of these groups are working together in parallel. Needs
+and solutions are clearly communicated on a regular basis. And, because these
+lines of reponsibility are becoming blurred, members can come to shared
+understandings of how the system works through code. After all, they are now
+working as one team.
+
+### The Platform Operator
+
+The role of the SRE or platform operator shifts from one of being primarily
+focused on system uptime to one where the operator is an integral part of the
+development process itself. While the application teams will be focused on new
+feature development, SREs will focus on the reliability of the whole system.
+This will require upleveled skills from that of a classic system administrator
+role. Instead of a simple restart of a service, an SRE will now be analyzing
+performance metrics, debugging defects, and helping to build plans to ensure
+data integrity.
```

Comment:
We should add something here about advising how to design a reliable and fault-tolerant system. (ie input in the actual design and development phase)